### DISCUSSION TITLE
editing the contrast, exposure, white balance, etc of video capture

### DISCUSSION THREAD

**Discussion by MahnoorK1** [bonsai-rx/bonsai#1025] [Q&A] [answered]
Hello, 

Is there a way to edit the video capture so that the output is higher in contrast? 
Currently there are a lot of shadows so I want to tweak the settings.
![b15e942d-e0fe-4ade-9731-11e5cd57faed](https://user-images.githubusercontent.com/112672417/188339039-b0bd0fd5-300f-430c-b325-9729f78158a4.jpeg)

Once these settings have been adjusted, is there a way to set them as a preset or to use them again, or do we have to adjust them each time manually?

Thank you for any insight!

**Comment by bruno-f-cruz** [comment, marked as answer]
Depending on which camera you have, you might be able to adjust its settings via the `VideoCaptureDevice ` `CaptureProperties`:
![image](https://user-images.githubusercontent.com/7049351/188395635-371e2243-6d3c-4705-83ed-688ccdbe2b6b.png)
